### DISCUSSION TITLE
How to access models listed by my openai api compatible service

### DISCUSSION THREAD

**Discussion by theconnectionist** [open-webui/open-webui#4794] [Q&A]
I created a local service with openai api compatibility. The idea is to run my agent in the service and implement custom logic and return responses using openai api endpoints. I was able to successfully connect to the service and verified that it is indeed calling the /v1/models endpoint in my service. However, I don't know where to find the "models" listed by my service so an end user can select and use it.

When I go to the Models admin tab, I only see the ability to manage ollama service based models.

Bottom line is I want to have a custom AI agent implemented in my service, which is exposing openai compatible api, and have that agent be accessible as a "model" through WebUI. Is that even possible?
